capellambse/diagram/capstyle.py

```diff
@@ -320,6 +324,11 @@ class in the form::
             "stroke-dasharray": "5",
             "text_fill": COLORS["dark_purple"],
         },
+        "Edge.PhysicalLink": {
+            "stroke": COLORS["red"],
+            "stroke-width": 2,
+            "text_fill": COLORS["red"],
+        },
```

Comment:
Please keep this dict sorted alphabetically. That's the only sliver of sanity we can cling to.